src/data/nat/factorial.lean

```diff
@@ -123,4 +123,24 @@ begin
   { exact add_factorial_lt_factorial_add _ hi }
 end
 
+lemma add_factorial_le_factorial_add (i : ℕ) (n : ℕ) :
```

Comment:
Maybe `add_factorial_succ_le_facrorial_add_suxx`, then you can use this name for the other lemma

Reply:
If you prefer, I can switch the primes in the names.  The actual lemma that I use if the one with the `succ`!

Reply:
Changed the name!